test/e2e/pages/vl-map.page.js

```diff
@@ -31,12 +31,23 @@ class VlMapPage extends Page {
     return this._getMap('#map-fullscreen');
   }
 
-  async load() {
-    await super.load(Config.baseUrl + '/demo/vl-map.html');
+  async load(url) {
+    await super.load(url || Config.baseUrl + '/demo/vl-map.html');
   }
 
   async _getMap(selector) {
-    return new VlMap(this.driver, selector);
+    const map = await new VlMap(this.driver, selector);
+    await this.driver.wait(async () => {
+      let resolved = false;
+      const promise = this.driver.executeScript('return arguments[0].ready', map);
+      try {
+        await this.driver.wait(async () => await promise.then(() => resolved = true), 1000);
+        return resolved;
```

Comment:
zou hier misschien gewoon true teruggeven. Als ik het goed lees gaat dat altijd true zijn tenzij we voorbij de 1 seconden wachten gaan en dan faalt de test, niet?

Reply:
@SanderKleykens mijn comment lijkt hier overeen te komen met de jouwe. Vreemd dat ik die niet gezien heb in het overzicht bij het reviewen

Reply:
Kan zijn dat je nog niet refreshed had sinds mijn comment ofzo :)